root/static/scripts/common/utility/releaseLabelKey.js

```diff
@@ -15,7 +15,7 @@ export default function releaseLabelKey(releaseLabel: ReleaseLabelT): string {
   let result = '';
   const label = ko.unwrap(releaseLabel.label);
   if (label) {
-    result += (label.id || '');
+    result += (label.id || nonEmpty(label.name) ? 'name-' + label.name : '');
```

Comment:
`||` has higher precedence than `?` so I'm not sure this is what was intended, and probably indicates a test is missing (same label name but different ids?)
```suggestion
    result += (label.id || (nonEmpty(label.name) ? 'name-' + label.name : ''));
```

Reply:
Good point. Added an extra test.